Src/Newtonsoft.Json-for-Unity.Tests/Assets/Newtonsoft.Json.Tests.Aot/EnsureTypes.cs

```diff
@@ -32,6 +32,15 @@ public void TestInitialize()
             AotHelper.EnsureList<KeyValuePair<string, string>>(); // used in Issue1322
             AotHelper.EnsureDictionary<string, decimal>(); // used in JsonSerializerTest.DeserializeDecimalDictionaryExact
             AotHelper.EnsureDictionary<string, int>(); // used in JsonSerializerCollectionsTests.SerializeCustomReadOnlyDictionary
+            AotHelper.EnsureDictionary<string, object>(); // used in combination with JsonExtensionDataAttribute, for example in SerializeExtensionData.CustomerInvoice
+
+            // used in Newtonsoft.Json.Tests.Serialization.TraceWriterTests.DeserializeMissingMemberConstructor
+            AotHelper.Ensure(() => {
+                // to not strip MajorRevision & MinorRevision
+                var version = new Version(1, 2, 3, 4);
+                var majorRev = version.MajorRevision;
+                var minorRev = version.MinorRevision;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'minorRev'.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity/pullRequest?prid=4755183)